src/tm_mad/ceph/clone

```diff
@@ -54,6 +54,7 @@ SRC_PATH=`arg_path $SRC`
 
 DISK_ID=$(echo $DST|awk -F. '{print $NF}')
 RBD_DST="${SRC_PATH}-${VM_ID}-${DISK_ID}"
+DST_DS_ID=$(echo $DST | sed s#//*#/#g | awk -F/ '{print $(NF-2)}')
```

Comment:
Maybe better like this:
`echo $DST | $SED 's#//*#/#g' | $AWK -F/ '{print $(NF-2)}'`

Reply:
I have written it consistently with the rest of the script - `awk` is used on line 55 `DISK_ID=$(echo $DST|awk -F. '{print $NF}')`

Should I change it there as well? What about other ceph driver scripts?

Reply:
We should definitely change that, but since we need to do it, you can leave it as is.